pkgs/development/python-modules/xdot/default.nix

```diff
@@ -1,19 +1,25 @@
-{ lib, buildPythonPackage, fetchPypi, isPy3k
-, wrapGAppsHook, gobject-introspection, pygobject3, graphviz, gtk3 }:
+{ lib, buildPythonPackage, fetchFromGitHub, isPy3k
+, wrapGAppsHook, gobject-introspection, pygobject3, graphviz, gtk3
+, numpy, pytestCheckHook }:
 
 buildPythonPackage rec {
   pname = "xdot";
   version = "1.2";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "3df91e6c671869bd2a6b2a8883fa3476dbe2ba763bd2a7646cf848a9eba71b70";
+  src = fetchFromGitHub {
```

Comment:
I switched this because the version on pypi does not include the `test.py` script.